cookbooks/aws-parallelcluster-config/recipes/base.rb

```diff
@@ -83,11 +83,16 @@
 end
 
 # Install parallelcluster specific supervisord config
+region = node['cluster']['region']
 template '/etc/parallelcluster/parallelcluster_supervisord.conf' do
   source 'base/parallelcluster_supervisord.conf.erb'
   owner 'root'
   group 'root'
   mode '0644'
+  variables(
+    region: region,
```

Comment:
why setting region? it should be avail through node['cluster']['region'] inside the .erb file

Reply:
This is true. I did this way to mainly to avoid duplication of node['cluster']['region'] within the ERB file and making it more readable. You think it's better to avoid this?

Reply:
As an alternative we may declare variables directly within the ERB file like this

```
<% @region = node['cluster']['region'] %>
```

Reply:
You think it's preferable?

Reply:
Talked with Luca about this. Both ways are ok.
Will keep the current approach.